script/collect.ts

```diff
@@ -90,7 +90,9 @@ async function fetchApiData() {
     )
   }
 
-  const response = await got(asset.browser_download_url, { json: true })
+  const response = await got(asset.browser_download_url, {
+    responseType: 'json',
```

Comment:
Good to know this API has changed. 

https://github.com/sindresorhus/got#responsetype

Reply:
Yes, see this but I change these to `responseType: json` since `.json()` returns the only body and type `unknown`.

The `unknown` anyway returns from the `responseType` so it no makes sense to use the previous API, I change these to `json()`.